docs/user/Alerts.md

```diff
@@ -0,0 +1,64 @@
+## MachineWithoutValidNode
+Each machine should have a valid node reference and one or more machines does not after 10 minutes.
+
+### Query
+```
+# for: 10m
+(mapi_machine_created_timestamp_seconds unless on(node) kube_node_info) > 0
+```
+
+### Possible Causes
+* The machine never became a node and joined the cluster
+* The node was deleted from the cluster via the api, but the machine still exists
+
+### Resolution
+If the machine never became a node, consult the machine troubleshooting guide.
+If the node was deleted from the api, you may choose to delete the machine object, if appropriate.  (FYI, The machine-api will automatically delete nodes, there is no need to delete node objects directly)
```

Comment:
i like the fyi, we cannot state this point enough times!

Reply:
I though it was the cloud provider controller (in kube-controller-manager) that does this? Not machine-api